src/components/profile/Profile.jsx

```diff
@@ -0,0 +1,50 @@
+import styles from "./Profile.module.css";
+import ProfileDetails from "./profileDetails/ProfileDetails";
```

Comment:
Package name is capitalized. Would prefer to suggest using lowercase but do not know how to do that in GitHubs Web-Review tool. So here is a pointer 🔽 
```suggestion
import ProfileDetails from "./ProfileDetails/ProfileDetails";
```
On my machine it does not work as expected without changing this path first.